conference.js

```diff
@@ -662,12 +662,14 @@ export default {
                     oldOnUnhandledRejection(event);
             };
         }
-
         return JitsiMeetJS.init(
-            Object.assign(
-                {enableAnalyticsLogging: analytics.isEnabled()}, config)
+            Object.assign({
+                enableAnalyticsLogging:
```

Comment:
Since we are here, is there a reason why this is a "fake" setting? This can be computed by lib-jitsi-meet inside JitsiMeetJS.init. Then we'd only need to do `JitsiMeetJS.init(config)` here, which looks a lot cleaner.

Also, as we stand, the mobile part would need an adjustment here: https://github.com/jitsi/jitsi-meet/blob/ec881e0fd05697efecb1096cee76a3af4a6bf6b8/react/features/base/lib-jitsi-meet/actions.js#L53 by doing what I suggest above we'd be bringing both closer and the code would be a bit simpler AFAIS.